integration/tests/cook/util.py

```diff
@@ -1045,3 +1059,24 @@ def active_pools(cook_url):
     """Returns the list of all active pools that exist"""
     pools, resp = all_pools(cook_url)
     return [p for p in pools if p['state'] == 'active'], resp
+
+
+@functools.lru_cache()
+def _cook_estimated_completion_constraint():
+    """Get the estimated completion constraint config from the /settings endpoint"""
+    cook_url = retrieve_cook_url()
+    _wait_for_cook(cook_url)
+    estimated_completion_constraint = settings(cook_url).get('estimated-completion-constraint', None)
+    logger.info(f"Cook's estimated completion constraint is {estimated_completion_constraint}")
+    return estimated_completion_constraint
+
+
+def has_ephemeral_hosts():
```

Comment:
Should we cache this result to avoid repeated invocations?

Reply:
I'd prefer not to cache this, it's just looking up an env var in test setup. Seems like overkill to me.